packages/remirror__extension-callout/src/callout-extension.ts

```diff
@@ -15,12 +15,16 @@ import {
   NodeExtensionSpec,
   nodeInputRule,
   NodeSpecOverride,
+  NodeViewMethod,
   omitExtraAttributes,
+  ProsemirrorNode,
   toggleWrap,
 } from '@remirror/core';
 import { Fragment, Slice } from '@remirror/pm/model';
 import { TextSelection } from '@remirror/pm/state';
+import { EditorView } from '@remirror/pm/view';
 
+import { ExtensionCalloutTheme } from '../../remirror__core/node_modules/@remirror/theme/src';
```

Comment:
```suggestion
import { ExtensionCalloutTheme } from '@remirror/theme';
```

You may need to add `@remirror/theme` to `packages/remirror__extension-callout/package.json`.